Backend/Models/PharmacyEntity/DTO/GetPharmaciesDTO.cs

```diff
@@ -1,7 +1,8 @@
-﻿using Newtonsoft.Json;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
+using Backend.Models.Common;
+using Newtonsoft.Json;
 
```

Comment:
Should this be deleted? This is not used anywhere and was changed by GetListDTO<T>

Reply:
You are correct. I already addresed this change in another branch and it will be fixed with the next MR